packages/terra-form-checkbox/translations/de.json

```diff
@@ -1,3 +1,4 @@
 {
-  "Terra.form.field.optional": "(Optional)"
+  "Terra.form.field.optional": "(Optional)",
```

Comment:
Is Optional really Optional in German?

Reply:
<img width="679" alt="Screen Shot 2019-06-13 at 12 24 41 PM" src="https://user-images.githubusercontent.com/1588747/59453783-4617e680-8dd6-11e9-9d42-d754efd1a163.png">

Reply:
Updated: https://github.com/cerner/terra-core/pull/2398/commits/e8962b5db5cd910886d9997d2e547d0d5b96f487